src/Compilers/CSharp/Test/Emit/Emit/EndToEndTests.cs

```diff
@@ -43,6 +43,23 @@ private static void RunInThread(Action action)
             }
         }
 
+        [Fact]
+        public void EmitWithSuppressedWarnAsError()
```

Comment:
>EmitWithSuppressedWarnAsError [](start = 20, length = 29)

so far, we only have two tests in this file, which are related to depth of code. Consider looking for another test file to host this test.